doc/whats_new/v1.3.rst

```diff
@@ -467,6 +467,12 @@ Changelog
   when `n_neighbors` is large and `algorithm="brute"` with non Euclidean metrics.
   :pr:`24076` by :user:`Meekail Zain <micky774>`, :user:`Julien Jerphanion <jjerphan>`.
 
+- |Enhancement| :meth:`neighbors.KNeighborsClassifier.predict` and
```

Comment:
This change needs to be included in the "Changed Model" section. There could be pipelines that worked before that will raise an error with this PR.